tests/simple/testdata/macros.textproto

```diff
@@ -1,2 +1,50 @@
 name: "macros"
 description: "Tests for CEL macros."
+section {
+  name: "exists"
+  description: "Checks for the existence of a map key or list element"
+  test {
+    name: "list_elem_exists"
+    expr: "[1, 2, 3].exists(e, e == 2)"
+    value: { bool_value: true }
+  }
+  test {
+    name: "not_list_elem_exists"
+    expr: "![1, 2, 3].exists(e, e > 3)"
+    value: { bool_value: true }
+  }
+  test {
+    name: "list_elem_mixed_type_exists"
+    expr: "[1, '2', 3].exists(e, e != '3')"
+    value: { bool_value: true }
+  }
+  test {
+    name: "list_elem_mixed_type_exists_error"
+    expr: "[1, '2', 3].exists(e, e == '3')"
```

Comment:
I see this is a similar behavior as with `elem_in_mixed_type_list_error`. If this list only had Strings, would the result be `false`?